projects/habitat_ovmm/configs/agent/floorplanner_eval.yaml

```diff
@@ -16,8 +16,10 @@ ENVIRONMENT:
   max_depth: 10.0         # maximum depth for depth sensor (in metres)
 
 AGENT:
-  max_steps: 500         # maximum number of steps before stopping an episode
+  max_steps: 10000         # maximum number of steps before stopping an episode; a lower value set for habitat episode termination 
```

Comment:
This is done to ensure termination on timeout does not come from the objectnav_agent. This would make it difficult to distinguish timeout from the case when the agent wants to terminate. Currently an episode is run for a maximum of 1250 steps and not 10000 steps. 1250 is set from the habitat configs. But I think we need to handle different termination modes from planner differently (eg. termination due to planning failure). I will leave this comment unresolved and get back to this in a later PR.